ledger/participant-integration-api/src/main/scala/platform/store/appendonlydao/events/TransactionsReader.scala

```diff
@@ -548,19 +548,19 @@ private[appendonlydao] object TransactionsReader {
           s"You can only split a range in a strictly positive number of chunks ($numberOfChunks)"
         )
 
-      case _ if numberOfChunks == 1 || minChunkSize >= rangeSize =>
+      case _ if numberOfChunks == 1 || minChunkSize > (rangeSize / 2L) =>
```

Comment:
Do we need this case actually? Methinks `case _` achieves exactly the same outcome

Reply:
Not quite. For `rangeSize == 100` and `minChunkSize = 51`, `case _` throws. I've seen it in action :trollface: